code/modules/deathmatch/deathmatch_modifier.dm

```diff
@@ -0,0 +1,478 @@
+					var/obj/item/implant/explosive/deathmatch/implant = new()
+					implant.implant(to_spawn, silent = TRUE, force = TRUE)
+			podspawn(list(
+				"path" = drop_pod_type,
+				"target" = to_strike,
+				"spawn" = to_spawn,
+			))
+			pod_spawned = TRUE
+			yet_to_spawn--
+			if(yet_to_spawn == 0)
+				break
+
+	if(pod_spawned)
+		COOLDOWN_START(src, drop_pod_cd, rand(interval[1], interval[2]))
+
+/datum/deathmatch_modifier/drop_pod/proc/populate_contents()
+	contents = typesof(/mob/living/basic/trooper/syndicate)
+
+/datum/deathmatch_modifier/drop_pod/monsters
+	name = "Drop Pod: Monsters"
+	description = "Monsters are raining from the sky!"
+
+/datum/deathmatch_modifier/drop_pod/monsters/populate_contents()
+	contents = list(
+		/mob/living/basic/ant = 2,
+		/mob/living/basic/construct/proteon = 2,
+		/mob/living/basic/flesh_spider = 2,
+		/mob/living/basic/garden_gnome = 2,
+		/mob/living/basic/killer_tomato = 2,
+		/mob/living/basic/leaper = 1,
+		/mob/living/basic/mega_arachnid = 1,
+		/mob/living/basic/mining/goliath = 1,
+		/mob/living/basic/mining/ice_demon = 1,
+		/mob/living/basic/mining/ice_whelp = 1,
+		/mob/living/basic/mining/lobstrosity = 1,
+		/mob/living/basic/mining/mook = 2,
+		/mob/living/basic/mouse/rat = 2,
+		/mob/living/basic/spider/giant/nurse/scrawny = 2,
+		/mob/living/basic/spider/giant/tarantula/scrawny = 2,
+		/mob/living/basic/spider/giant/hunter/scrawny = 2,
+		/mob/living/simple_animal/hostile/dark_wizard = 2,
+		/mob/living/simple_animal/hostile/retaliate/goose = 2,
+		/mob/living/simple_animal/hostile/ooze = 1,
+		/mob/living/simple_animal/hostile/vatbeast = 1,
+	)
+
+/datum/deathmatch_modifier/drop_pod/missiles
+	name = "Drop Pod: Cruise Missiles"
+	description = "You're going to get shelled hard"
+	drop_pod_type = /obj/structure/closet/supplypod/deadmatch_missile
+	interval = list(3 SECONDS, 5 SECONDS)
+	amount = list(1, 2)
+
+/datum/deathmatch_modifier/drop_pod/missiles/populate_contents()
+	return
+
+/datum/deathmatch_modifier/explode_on_death
+	name = "Explosive Death"
+	description = "Everyone gets a microbomb that cannot be manually activated."
+
+/datum/deathmatch_modifier/explode_on_death/on_start_game(datum/deathmatch_lobby/lobby)
+	ADD_TRAIT(lobby, TRAIT_DEATHMATCH_EXPLOSIVE_IMPLANTS, DEATHMATCH_TRAIT)
+
+/datum/deathmatch_modifier/explode_on_death/apply(mob/living/carbon/player, datum/deathmatch_lobby/lobby)
+	var/obj/item/implant/explosive/deathmatch/implant = new()
+	implant.implant(player, silent = TRUE, force = TRUE)
+
+/datum/deathmatch_modifier/helgrasp
+	name = "Helgrasped"
+	description = "Cursed hands are being thrown at you!"
+
+/datum/deathmatch_modifier/helgrasp/apply(mob/living/carbon/player, datum/deathmatch_lobby/lobby)
+	var/metabolism_rate = /datum/reagent/inverse/helgrasp/heretic::metabolization_rate
+	player.reagents.add_reagent(/datum/reagent/inverse/helgrasp/heretic, initial(lobby.map.automatic_gameend_time) / metabolism_rate)
```

Comment:
We really need to make this a status effect which is _also_ applied by the reagent istg
Not in this PR though